regression/regression_utils.py

```diff
@@ -31,6 +32,38 @@
 """
 
 
+class RetryTestsMetaclass(type):
+
+    NUM_RETRIES = 2
+    FLAKY_ERROR_CLASSES = (AssertionError,)
```

Comment:
I just linked to one above that would have succeeded had it been restarted. I can try to dig some up from the Game of Thrones entities too, but we've seen occasional failures where those queries don't return the expected output (even though the entities never change).